go/chat/inboxsource.go

```diff
@@ -898,6 +873,12 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 		msgs, err := s.G().ConvSource.GetMessagesWithRemotes(ctx, conversationRemote.Metadata.ConversationID,
 			uid, conversationRemote.MaxMsgs, conversationRemote.Metadata.FinalizeInfo)
 		if err != nil {
+			convErr := s.checkRekeyError(ctx, err, conversationRemote, unverifiedTLFName)
```

Comment:
@mlsteele I noticed this was missing from this branch, it is right to add it back in right?